src/metabase/lib/js.cljs

```diff
@@ -1178,3 +1179,11 @@
   "Add or update a filter against `temporal-column`. Modify the temporal unit for any breakouts."
   [a-query temporal-column stage-number start end]
   (lib.core/update-temporal-filter a-query temporal-column stage-number start end))
+
+(defn ^:export assignable?
```

Comment:
I'm afraid this can lead to confusion given that we already have `metabase.types/assignable?`.

Reply:
I changed the name.